nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/resources/test-export-alias-config.xml

```diff
@@ -0,0 +1,8 @@
+<component name="org.nuxeo.ecm.automation.export.alias.test"
+  version="1.0">
+
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+    <property name="nuxeo.automation.export.aliases">true</property>
+  </extension>
```

Comment:
Agreed, an extension point with the default value and a `<documentation>` element should be deployed as part of the regular bundle.